src/features/select-book/selectBookSlice.ts

```diff
@@ -65,7 +65,18 @@ export const {
 } = selectBookSlice.actions;
 
 export const selectAvailableBooks = (state: RootState) => {
-  return state.content.books;
+  //define and filter array of viewable books
+  let ViewableBooks: Array<IBook> = state.content.books.filter( book => 
```

Comment:
You don't really have to declare a variable here and then return it at line `79`. You could instead simply add a return statement to the front of this:


```ts
return state.content.books.filter(/* ... */)
```

and the filter call can still span multiple lines as you have now.